### PR TITLE
System_Runtime_GetAddressVersion

### PR DESCRIPTION
**Pull request by devhawk**
Adds a System_Runtime_GetAddressVersion interop method

### CODE REVIEW

**Review by shargon — APPROVED** [review]

**Comment by Liaojinghui** [comment]
Need to add an interface to the compiler

**Review by Liaojinghui — APPROVED** [review]

File: src/neo/SmartContract/ApplicationEngine.Runtime.cs [hunk 1]
```diff
@@ -46,6 +46,12 @@ partial class ApplicationEngine
         /// </summary>
         public static readonly InteropDescriptor System_Runtime_GetNetwork = Register("System.Runtime.GetNetwork", nameof(GetNetwork), 1 << 3, CallFlags.None);
 
+        /// <summary>
+        /// The <see cref="InteropDescriptor"/> of System.Runtime.GetNetwork.
```
Comment (Liaojinghui) [inline src/neo/SmartContract/ApplicationEngine.Runtime.cs:50]: Please update the comment

File: src/neo/SmartContract/ApplicationEngine.Runtime.cs [hunk 2]
```diff
@@ -46,6 +46,12 @@ partial class ApplicationEngine
         /// </summary>
         public static readonly InteropDescriptor System_Runtime_GetNetwork = Register("System.Runtime.GetNetwork", nameof(GetNetwork), 1 << 3, CallFlags.None);
 
+        /// <summary>
+        /// The <see cref="InteropDescriptor"/> of System.Runtime.GetAddressVersion.
+        /// Gets the magic number of the current network.
```
Comment (Liaojinghui) [inline src/neo/SmartContract/ApplicationEngine.Runtime.cs:51]: Well, appearently you forget to update this line~~~~~~

Comment (Liaojinghui) [inline src/neo/SmartContract/ApplicationEngine.Runtime.cs:51]:
```suggestion
        /// Gets the address version.
```

**Comment by devhawk** [comment]
@shargon why was the PR closed without merging?

**Comment by Liaojinghui** [comment]
cause shargon deleted the `develop` branch, by mistake i guess~~~

**Comment by shargon** [comment]
When I merged this one it was closed, my bad https://github.com/neo-project/neo/pull/2682

**Pull request closed without merging**